src/app/login/login.component.scss

```diff
@@ -1,24 +1,19 @@
-.card-row {
```

Comment:
There are lots of arbitrary margins here. I'm not sure if this is on purpose. If the design of the page is not part of this task, then please add a todo here to fix this. Ideally we should use flexbox to position things and then use margins only to give some kind of uniform space

Reply:
The below CSS on the root div in login component seems to accomplish almost the same thing as what the current UI looks like

```css
display: flex;
justify-content: space-around;
align-items: center;
height: 50%; // Can be altered as per requirement later
```

Reply:
I can probably work with you on this. I will add a TODO for now.